modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/paths/APIBuilder.path

```diff
@@ -0,0 +1,61 @@
+<html>
+<head>
+<title>APIBuilder</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+<tr><td rowspan="1" colspan="3">APIBuilder</td></tr>
+</thead>
+
+<tbody>
+<tr>
+	<td>APPLICATION_STATUS</td>
+	<td>//div[@class='table-list-title' and contains(.,'${key_title}')]/../..//div[contains(.,'${key_status}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CANCEL_BUTTON</td>
```

Comment:
There are a few "Cancel" buttons in `Button.path` file, is there any one works? Could you help check if we can use any existing ones so that we can delete from our APIBuilder.path?

Reply:
good point Ashley!